### DISCUSSION TITLE
Couple MacOS issues, trying to ensure I'm not doing something dumb before filing an issue

### DISCUSSION THREAD

**Discussion by TheCleric** [wezterm/wezterm#694] [Q&A] [answered]
On MacOS Catalina, I installed wezterm for the first time today. Here's a few things that happened:

- It installed, but I was prompted to update CommandLineTools (not a big deal, and understandable, just noting in case it relates to the other items)
- After installing the CommandLineTools update (which I had to do via the remove and reinstall method as no updates were found), I did not have a wezterm app in my applications. 
- I was able to find it in `/usr/local/Cellar/wezterm/20210405-110924/WezTerm.app` and launched it
- Some of the app functionality does not seem to work such as the menu item in the menu bar for WezTerm does nothing, and Cmd+K for clearing the screen doesn't seem to work.

Is any of this expected, or should I file a bug?

**Comment by wez** [comment, marked as answer]
The command line tools thing sounds like homebrew itself and not part of wezterm.
When using homebrew, there isn't a well-supported way for a tap to install the .app to /Applications.
https://github.com/wez/wezterm/issues/501 is tracking that, but no one is working on resolving it.

WezTerm doesn't have menu items in the menubar, as the menubar is not a concept that applies to all platforms.  https://github.com/wez/wezterm/issues/162 is an existing feature request for that.

Regarding CMD-k, the default assignment for that key is `ClearScrollback="ScrollbackOnly"`, which deletes the scrollback but preserves the current screen contents.   You can rebind it to clear both the viewport and the scrollback if you wish: https://wezfurlong.org/wezterm/config/lua/keyassignment/ClearScrollback.html

**Reply by TheCleric** [reply]
Thanks! So it seems like all of this is normal. Glad I didn't completely waste your time by submitting an issue. Thanks for the followup!
